src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
-'use strict';
+"use strict";
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```
This way is better so you do not need to assign healt in each class.

Reply:
@ysrckr I am a bit confused here, do we really need these assignments here? Because as I understand, we already reaching these assignments by extending the Herbivore class to the Animal class. Or am I missing something?

Reply:
You need to take in the `name` in Animal Class. You don't need to take in the health, you can just assign it to 100. In herbivore, we need to take in the name and pass that to the super. I hope it cleared it out for you.